README.md

```diff
@@ -1,136 +1,5 @@
-
-Since the DataDome framework retains the cookies after resolving the captcha, this test can be done only the first time you used the BLOCKUA user agent. To reproduce the test case, you can use the following code snippet to manually clear the cookies stored in your app
-
-```swift
-for cookie in HTTPCookieStorage.shared.cookies ?? [] {
-	HTTPCookieStorage.shared.deleteCookie(cookie)
-}
-```
-
-## Disable method swizzling
-The DataDome Framework is using method swizzling to setup a `proxy` listening to all `URLRequests` performed by the app. The framework will then detect automatically blocked requests, lookup dynamically the top view controller in your app hierarchy and attach a captcha view controller to it. Once resolved, the framework will dismiss the captcha view controller and retry the blocked request with a new generate cookie. All this is done under the hood.
-
-If you want to disable the method swizzling and manually tell the framework which request to monitor instead of monitoring all the traffic or / and you want to manually present and dismiss the captcha controller, please consider the following steps:
-
-1. In your Info.plist, add a new entry with Boolean type, use **DataDomeProxyEnabled** as key and **NO** as value. This will disable method swizzling in the framework.
-2. For each request to protect with DataDome, use the following methods to create the data task
-
-```swift
-func protectedDataTask(with url: URL,
-					 captchaDelegate: CaptchaDelegate?,
-					 completionHandler: @escaping (Data?, URLResponse?, Error?) -> Void) 
--> URLSessionDataTask
-```
-
-```swift
-func protectedDataTask(with request: URLSessionRequest,
-					 captchaDelegate: CaptchaDelegate?,
-					 completionHandler: @escaping (Data?, URLResponse?, Error?) -> Void) 
--> URLSessionDataTask
-```
-
-Only the `URLSessionDataTasks` created by the above two methods are monitored and protected by DataDome. 
-
-Here a sample code to create and execute a data task
-
-```swift
-import DataDomeSDK
-
-
-guard let url = URL(string: "https://datadome.co") else {
-	return
-}
-
-let task = URLSession.shared.protectedDataTask(with: url, captchaDelegate: self, completionHandler: { _, _, _ in
-	print("Request completed")
-})
-        
-task.resume()
-```
-
-
-To manually manage the presentation of the captcha view controller, implement the `CaptchaDelegate` protocol
-
-```swift
-extension MyViewController: CaptchaDelegate {
-    func present(captchaController controller: UIViewController) {
-        self.navigationController?.present(controller, animated: true, completion: nil)
-    }
-    
-    func dismiss(captchaController controller: UIViewController) {
-        controller.dismiss(animated: true, completion: nil)
-    }
-}
-```
+This is a mirror dedicated to disctrubution of the DataDomeSDK for SPM.
```

Comment:
do you really mean "mirror" or minor? 😅
Please change disctrubution to distribution
Also I would rather put Swift Package Manager instead of SPM